doc/source/refs.bib

```diff
@@ -960,3 +961,17 @@ @article{Cacciari:2005ry
     pages = "034",
     year = "2005"
 }
+
+@article{Bierenbaum:2022biv,
```

Comment:
Can you please also add one or two sentences to the docs? e.g. here: https://github.com/NNPDF/eko/blob/661a1387f09ffd4ff68e0c15972e929b5031d8f0/doc/source/theory/Matching.rst?plain=1#L34

Reply:
Sure I can take care of it!!

Reply:
addressed in 3dd1d9558bd3a388e8cd788294be744196676ee5